appium/webdriver/webdriver.py

```diff
@@ -341,6 +341,30 @@ def find_elements_by_android_uiautomator(self, uia_string):
         """
         return self.find_elements(by=By.ANDROID_UIAUTOMATOR, value=uia_string)
 
+    def find_element_by_android_viewtag(self, tag):
+        """Finds element by tag in Android.
```

Comment:
Perhaps, it worth to mentions that such locator only works for Espresso driver

Reply:
:+1: Yeah, I'm appending `find_element_by_android_viewtag` section in README. I'll mention the point here as well.